Docs/Configuration.tex

```diff
@@ -265,35 +265,35 @@ \subsection{Configuration Terms}\label{configuration-terms}
   \texttt{invalid\ value} if missing.
 \item
   \texttt{fatal\ behaviour} --- behaviour leading to boot termination.
-  Implementation must stop the boot process from going any further until
-  next host system boot. It is allowed but not required to perform cold
-  reboot or show any warning message.
+  Implementations must prevent the boot process from proceeding until the
+  next host system boot. It is permitted, but not required, to execute
+  cold reboots or to show warning messages in such cases.
 \item
   \texttt{undefined\ behaviour} --- behaviour not prescribed by this
-  document. Implementation is allowed to take any measures including but
-  not limited to \texttt{fatal\ behaviour}, assuming any states or
-  values, or ignoring, unless these measures negatively affect system
-  security in general.
+  document. Implementations are permitted to take any measures including, but
+  not limited to, \texttt{fatal\ behaviour}, assumptions of any state or
+  value, or disregarding the state or value. This is however subject to such
+  measures not negatively impacting upon system integrity.
 \end{itemize}
 
 \subsection{Configuration Processing}\label{configuration-processing}
 
-\texttt{OC\ config} is guaranteed to be processed at least once if it
-was found. Depending on OpenCore bootstrapping mechanism multiple
-\texttt{OC\ config} files may lead to reading any of them. No
-\texttt{OC\ Config} may be present on disk, in which case all the values
-read follow the rules of \texttt{invalid\ value} and
-\texttt{optional\ value}.
-
-\texttt{OC\ config} has size, nesting, and key amount limitations.
-\texttt{OC\ config} size does not exceed \texttt{16\ MBs}.
-\texttt{OC\ config} has no more than \texttt{8} nesting levels.
-\texttt{OC\ config} has up to \texttt{16384} XML nodes (i.e.~one
+The \texttt{OC\ config} file is guaranteed to be processed at least once if found.
+Depending on OpenCore bootstrapping mechanism, the presence of multiple
+\texttt{OC\ config} files may lead to the reading any of them. It is permissible that
+no \texttt{OC\ Config} file is present on disk. In such cases, all the values
+read shall follow the rules of \texttt{invalid\ values} and
+\texttt{optional\ values}.
```

Comment:
Suggest: "It is permissible for no config file to be present on disk. In such cases, all values will follow the rules for ..." (NB that includes not saying "values read", which if I understand correctly, I think would be specifically wrong to say here.)

Reply:
Concur